.travis.yml

```diff
@@ -36,4 +36,5 @@ script:
  - ./.buildifier.sh
  - python -m unittest discover -s reconciletags -p 'reconciletags_test.py'
  - bazel clean && bazel build //...
+ - sudo rm -rf /etc/docker
```

Comment:
You do have to remove the entire folder (at least from my experience with it locally).